lib/chewy/search/scrolling.rb

```diff
@@ -34,6 +34,9 @@ def scroll_batches(batch_size: Request::DEFAULT_BATCH_SIZE, scroll: Request::DEF
         scroll_id = nil
 
         loop do
+          failures = result.dig('_shards', 'failures')
+          raise Chewy::Error, failures if failures.present?
```

Comment:
I think `Chewy::Error` is not the best thing here, from your example, `failures` is a hash, and `Chewy::Error` is simply a StandardError with a string argument. Please make a specific error message and convert `failure` into a meaningful string? For instance https://github.com/toptal/chewy/blob/10ff43b6c4f213cb215267630c01b02a393197f1/lib/chewy/errors.rb#L20-L32